foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -74,7 +74,7 @@ GitHub allows you to publish web projects directly from a GitHub repository. Doi
 There are a couple of ways to go about doing this, but the simplest is this:
 
 - make sure that the main HTML file of your project is called `index.html`. If it is not, you will need to rename it.
-- go to your GitHub repo on the web and click the **Settings** button.
+- go to your GitHub repo on the web and click the **Settings** button. The one from a dropdown menu in top right is NOT the one - that's user settings.
```

Comment:
What's the reasoning for adding verbiage about __where not to go__ as opposed to being more descriptive about __where to go__?

It's possible, based on the user's screen or viewport size, that the Settings button is in clear view in the tabs available below the repository's name **or** behind the atypical 3 dots icon representing what I'll refer to as "more options" here.

Reply:
@marvingay 
I understand this might be a little counter-intuitive but this is due to:
1) Somehow people get to user settings all the time. This makes sure they don't. This in turn should make it so they actually get where they should get.
2) As you noted, `Settings` is going to move depending on the viewport. User menu is going to be there in top right, always, so it can be referenced clearly.
3) I'm betting that it is more likely that the appearance or placement of `Settings` changes than that the user menu is going to change, which makes the instruction more resilient for whatever changes GitHub might do.

I also proposed those alternative solutions in the [Discord message](https://discord.com/channels/505093832157691914/540903304046182425/958100284184358942):
1. Include more verbose description of the Settings button, like:
>go to your GitHub repo on the web and click the Settings button that's on top of the file list

or

>go to your GitHub repo on the web and click the Settings button with a cog to the left of it

3. Include a screenshot of how the button looks like
4. Include an example URL for where the button should take the user

Given your observation about viewports, solutions 1 & 3 would also need a mention about three dots icon.

Another alternative solution would be including a picture of a repo page with `Settings` highlighted, not necessarily embedded in the instructions - could just be a link in this bullet point of the instructions:
>change the Source from none to main branch and click Save.

that'd say "Don't see anything like that? Check this picture with repo settings button highlighted".
Why there? Because that's when people realise they've gone to the wrong place and start asking.

Reply:
I'm personally a proponent of being more specific in directing them to where they need to go, as opposed to telling them where not to go, as the latter doesn't solve the actual problem if they still don't know where it is.

Solution 1 is likely the path of least resistance.

Reply:
@marvingay 
Alright, verbiage has been changed. On accident I discovered that the same instructions are repeated in the Landing Page project - should they be changed as well? I usually see people struggle with finding settings on the Recipes project, not Landing Page.